frameworks/helloworld/src/main/dist/svc.yml

```diff
@@ -7,6 +7,10 @@ pods:
     count: {{HELLO_COUNT}}
     placement: {{HELLO_PLACEMENT}}
     container:
+      rlimits:
```

Comment:
Two things:
1. Why are you adding this within the container element?  Regression testing?
2. Please don't use the default `svc.yml` file of `hello-world` to contain all use cases.  There is an `examples/` directory for exercising particular use cases.  Tests can then use those alternate scenario YAML files.